modules/core/test/v2/unit/ethWallet.ts

```diff
@@ -292,3 +292,35 @@ describe('Add final signature to ETH tx from offline vault', function() {
     response.txHex.should.equal(expectedResult.txHex);
   });
 });
+
+describe('prebuildTransaction', function() {
+  let bitgo;
+  let ethWallet;
+  let recipients;
+  let bgUrl;
+  let gasLimit;
+
+  before(function() {
+    bitgo = new TestBitGo({ env: 'test' });
+    bitgo.initializeTestVars();
+    const coin = bitgo.coin('teth');
+    ethWallet = coin.newWalletObject(bitgo, coin, {});
+    gasLimit = 2100000;
+    recipients = [{
+      address: '0xe59dfe5c67114b39a5662cc856be536c614124c0',
+      amount: '100000'
+    }];
+    bgUrl = common.Environments[bitgo.getEnv()].uri;
+    nock(bgUrl)
+      .post('/api/v2/teth/wallet/' + ethWallet.id() + '/tx/build', {
+        recipients,
+        gasLimit,
+      })
+      .reply(200, { success: true });
+  });
+
+  it('should successfully accept gasLimit as a param', co(function *() {
+    const prebuild = yield ethWallet.prebuildTransaction({ recipients, gasLimit });
+    prebuild.success.should.equal(true);
```

Comment:
This should check to make sure the nock was consumed successfully, since that is what you are really testing (param is correctly passed to server).

Reply:
the nock will throw an error if its not called with gasLimit

Reply:
are you suggesting I uses sinon and stubs?